ui/js/perf.js

```diff
@@ -59,6 +59,47 @@ perf.factory('PhSeries', ['$http', 'thServiceDomain', function($http, thServiceD
         });
     };
 
+    var _getPlatformList = function(projectName, timeRange) {
+        var platformURL = thServiceDomain + '/api/project/' + projectName +
+            '/performance-data/platforms/?interval=' +
+            timeRange;
+        return $http.get(platformURL).then(function(response) {
+            return {
+                platformList: response.data
+            };
+        });
+
+    };
+
+    var _getSeriesByPlatform = function(projectName, timeRange, platform, optionMap) {
+        var specifyPlatformURL = thServiceDomain + '/api/project/' + projectName +
+            '/performance-data/get_performance_series_summary/?interval=' +
+            timeRange + '&machine_platform=' + platform;
+
+        return $http.get(specifyPlatformURL).then(function(response) {
+            var seriesList = [];
+            var testList = [];
+
+            Object.keys(response.data).forEach(function(signature){
```

Comment:
Space between function(signature) and {

Also, I prefer `_.keys(response.data)`, as it's a bit more concise.